### PR TITLE
TimeCommand: changed behavior to affect a single world

### PR DESCRIPTION
**Pull request by Nerahikada**
## Introduction
Currently, the time command affects all loaded worlds.
This PR restricts the affect of the time command to a single world.

### Behavioural changes
- The time command no longer affects the time of all worlds.
  - The player executes the command, the time of the world the player is in will be changed.
  - The console executed the command, the default world time will be changed.

### CODE REVIEW

File: src/command/defaults/TimeCommand.php [hunk 1]
```diff
@@ -98,48 +95,30 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
...
-			}
+			$value = match($args[1]){
+				"day" => World::TIME_DAY,
+				"noon" => World::TIME_NOON,
+				"sunset" => World::TIME_SUNSET,
+				"night" => World::TIME_NIGHT,
+				"midnight" => World::TIME_MIDNIGHT,
+				"sunrise" => World::TIME_SUNRISE,
+				default => $this->getInteger($sender, $args[1], 0),
+			};
```
Comment (dktapps) [inline src/command/defaults/TimeCommand.php]: This change is unrelated.

**Review by dktapps — COMMENTED** [review]
I'm doubtful that this is desirable.

**Comment by xDidntPot** [comment]
Tbh this is useful (atleast for me).

File: src/command/defaults/TimeCommand.php (hunk 1 above)
Comment (Nerahikada) [reply src/command/defaults/TimeCommand.php]: I didn't think it was necessary to keep the old code.

Comment (dktapps) [reply src/command/defaults/TimeCommand.php]: It's also not necessary to change it, and doesn't contribute to the goal of the PR. Don't change unrelated things.

Comment (Nerahikada) [reply src/command/defaults/TimeCommand.php]: I don't understand why you would keep the old code if there is no purpose for it, such as backward compatibility. LOL

Comment (dktapps) [reply src/command/defaults/TimeCommand.php]: As I said, it's not adding to the goal of the PR. It's just polluting your diff.

**Comment by dktapps** [comment]
Closing due to lack of activity and lack of compelling reasons to make the change.
